packages/i18n-calypso/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 6.0.0
```

Comment:
Doing a major update because it's been two years since a major update, so I'm unsure what all has changed.